packages/devtools_app/lib/src/shared/service_extensions.dart

```diff
@@ -91,19 +91,20 @@ class ServiceExtensionDescription<T> {
 
   final List<String> displayValues;
 
-  final String gaScreenName; // Analytics screen (screen name where item lives).
+  final String?
+      gaScreenName; // Analytics screen (screen name where item lives).
 
-  final String gaItem; // Analytics item name (toggleable item's name).
+  final String? gaItem; // Analytics item name (toggleable item's name).
```

Comment:
formatting. For both of these, move the documentation to be above the class members and change to proper dart doc with three slashes ///

Reply:
done